asm/macros/event.inc

```diff
@@ -1932,10 +1932,10 @@
 	closebraillemessage
 	.endm
 
-	@ Creates an "event legal" Pokémon for an encounter
-	.macro seteventmon species:req, level:req, item=ITEM_NONE
+	@ Creates a Pokémon with the modernFatefulEncounter bit set for an encounter
+	.macro setmodernfatefulencountermon species:req, level:req, item=ITEM_NONE
 	setvar VAR_0x8004, \species
 	setvar VAR_0x8005, \level
 	setvar VAR_0x8006, \item
-	special CreateEventLegalEnemyMon
+	special CreateModernFatefulEncounterEnemyMon
 	.endm
```

Comment:
These should probably be `seteventmon` and `CreateEnemyEventMon`, it's more descriptive about what they're used for (it's just that event pokemon have the "modern fateful encounter" bit set).